src/main/java/com/algorand/algosdk/transaction/Lease.java

```diff
@@ -0,0 +1,70 @@
+package com.algorand.algosdk.transaction;
+
+import com.algorand.algosdk.util.Encoder;
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.Arrays;
+import java.util.Random;
+
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class Lease {
```

Comment:
Please describe what this class is all about. Also helps adding references to the algorand doc where the concept is presented.